data/coalition/coalition news

```diff
@@ -0,0 +1,177 @@
+			`"Some fresh produce would you like, Captain? If purchase these, you do, teach you a family recipe I could. I'm sure that love it, a human would. Probably."`
+			"After noticing you looking at the produce, the farmer asks if you would like some free samples, before grabbing an entire box with various fruits and vegetables, and handing it to you."
+			"Dozens of farmers in different stands call out to you as you walk by the spaceport, asking you to take a look at the special prices of today's 'food expo.' You later look into it, and turns out yesterday there also was an expo, as well as the day before that. Tomorrow's starts the moment today's ends."
+
+news "heliarch agent"
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Heliarch agent"
+	message
+		word
+			`"Protect and serve."`
+			`"If any assistance you need, at your disposal I am."`
+			`"If any trouble you find, report it to us you must."`
+			`"Saryd, Kimek, Arach, human, no matter what species, to protect all of our citizens committed we are."`
+			`"Enjoy my post well enough, I do, but still to get a break during the Coalition games to watch them, I am."`
+			`"Part of a Punisher crew, I am. Land, refuel, rest, and back to working on that magnificient ship I am!"`
+			`"Offered the position of elder, I actually have been. Only, not interested in that governing activity, I am."`
+			`"Apologies, but make haste I must. Gone missing some garrison guns have, and back to cleaning Stalker engines I will be, if find them I don't!"`
+			`"Difficult sometimes it is, to be a mediator. If careful with my words I am not, a Saryd might think an Arach talking about longcows has insulted their mother."`
+			"An agent is trying to calm down a small crowd outside a local bar. After some minutes the more agitated drunkards are convinced to accompany the agent to a nearby Heliarch office."
+
+news "heliarch candidate"
+	location
+		government "Heliarch"
+	name
+		word
+			"Heliarch candidate"
+	message
+		word
+			`"Laws, history, theory, too much of it I have read, and still behind I am..."`
+			`"As part of my trials, read the works of elder Elirom, I had to. Exceeded my expectations, his books have."`
+			`"To prove our dedication and show our potential, the trials we candidates go through are. Though difficult, we're sure to be apt to serve once passed, we have."`
+
+news "coalition ad"
+	location
+		government "Coalition"
+	name
+		word
+			"Advertisement"
+	message
+		word
+			"You see a commercial where a Saryd sits down after a workout, and as the camera spins it switches to a Kimek studying, and then to an Arach watching sports while cheering. It seems to be an ad for a brand of foldable chairs."
+			"A commercial for house Debdo plays on a local monitor, with various scenes of tourist spots being accompanied by an upbeat, serene jingle."
```

Comment:
Here's another implication I think is pretty interesting. The Houses aren't just political groups/families, they're also corporation-esque and advertise their services/products (in this case, tourism).